test/parallel/test-console-assing-undefined.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+// Should be above require, because code in require read console
+// what we are trying to avoid
+// set should be earlier than get
+
+global.console = undefined;
+
+// Initially, the `console` variable is `undefined`, since console will be
+// lazily loaded in the getter.
+
```

Comment:
Just don't assign it anywhere. Instead of `const common = require('../common');` do `require('../common');`.